easybuild/main.py

```diff
@@ -70,6 +70,7 @@
 from easybuild.tools.parallelbuild import submit_jobs
 from easybuild.tools.repository.repository import init_repository
 from easybuild.tools.testing import create_test_report, overall_test_report, regtest, session_state
+from easybuild.tools.build_from_specsfile import handle_specsfile
```

Comment:
rename:
`easybuild.tools.build_from_specsfile` -> `easybuild.tools.easystack` (and maybe it should be under `easybuild.framework` instead, next to `easyconfig` and `easyblock`?)

`handle_specsfile` -> `parse_easystack`